docker/compose/README.md

```diff
@@ -5,27 +5,31 @@ custom_edit_url: https://github.com/diem/diem/edit/master/docker/compose/README.
 ---
 
 This directory contains the following compose configurations:
-* **validator-testnet**: creates a single validator test network
-* **faucet**: creates a faucet that directly connects to validator-testnet
+* **validator-testnet**: creates a single validator test network, and a faucet that directly connects to it
+* **client**: creates a client which connects to the above validator/faucet
 
 To use these compositions:
 1. [Download](https://docs.docker.com/install/) and install Docker and Docker Compose (comes with Docker for Mac and Windows).
 2. Open your favorite terminal and enter the appropriate composition directory
 3. Run `docker-compose up`
 
 To build your own complete testnet:
-0. Review both **validator-testnet** and **faucet** docker-compose.yaml and ensure the image points to the same tagged image.
-1. Start the **validator-testnet**:
+1. Start the **validator-testnet** and **faucet**:
     1. Enter the **validator-testnet** directory `cd validator-testnet`
     2. Start the composition `docker-compose up -d`
-    3. Confirm that waypoint.txt is not empty
-    4. Return to the compose directory: `cd ..`
-2. Start **faucet**:
-    1. Enter the **faucet** directory: `cd faucet`
-    2. Copy the testnet waypoint: `cp ../validator-testnet/waypoint.txt .`
```

Comment:
I suppose we don't need waypoints anymore? @davidiw I'm not as familiar with waypoints as they are today so maybe you have some comments?

Reply:
If i'm reading correctly below, the validator will populate all of these things in a shared volume and the other docker images mount that shared volume to get access to these things. Its nice that we're able to stop needing these files checked into the repo.

Reply:
we don't need waypoints in general for non-verifying client.
waypoint can be used for validating server latest blocks, but will need a verifying client to handle the logic.
so for starting a test env in docker, there is no need.

Reply:
I think that the `cli` still ends up using the waypoint for one reason or another.

Reply:
even though there might not be a need for it, the cli currently requires it: 
```
error: The following required arguments were not provided:
     --waypoint <waypoint>
     --waypoint_url <waypoint_url>
```

in either case the waypoint does get populated by the validator and exists on the shared volume (for whatever might need it)

Reply:
Once you get the Rust SDK done, we should build a new CLI....

Reply:
The Faucet doesn't use the CLI any more afaiu, the old one did.

Reply:
the faucet doesn't use the CLI, but the CLI does use the faucet (I think I'm missing something)

Reply:
CLI does require a waypoint but it really doesn't need it (from an internal perspective).

Reply:
@davidiw I'll put up a PR after this to remove the requirement we pass one (I've been taking rust tutorials and am excited to see if I've learned anything :-P)